site/vocs.config.tsx

```diff
@@ -66,40 +65,27 @@ export default defineConfig({
   },
   rootDir: './docs/',
   sidebar,
-  socials: [
-    {
-      icon: 'github',
-      link: 'https://github.com/coinbase/onchainkit',
-    },
-  ],
   theme: {
     accentColor: {
       light: '#FF5D2F',
       dark: '#FF5D2F',
     },
+    variables: {
+      color: {
+        background: {
+          light: 'white',
+          dark: 'black',
+        },
+        text: {
+          light: '#373A3E',
+          dark: '#FFFFFF',
+        }
+      }
+    }
   },
   topNav: [
+    { text: 'Examples', link: '/examples' },
     { text: 'Docs', link: '/getting-started', match: '/docs' },
-    {
```

Comment:
Let's keep the navigation as it is...

We can update it maybe later, but for now those links are all super useful.

Reply:
reset the topNav to the original state